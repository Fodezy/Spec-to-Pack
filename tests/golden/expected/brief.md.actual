# Golden Test Spec
Create a deterministic template testing system