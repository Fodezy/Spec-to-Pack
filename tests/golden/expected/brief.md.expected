# Golden Test Spec

**Version:** 1.0.0

**Description:** Test specification for golden template tests


## Problem Statement

Create a deterministic template testing system


### Context

This system validates template consistency across versions


## Success Metrics


- _No success metrics defined_


## Constraints

- **Offline Mode:** Yes
- **Budget:** 80000 tokens
- **Duration:** 30 minutes

## Configuration

- **Audience:** balanced
- **Development Flow:** agile
- **Test Depth:** pyramid

---

*Generated with Spec-to-Pack Studio*
*Template: balanced-1.0.0*